netbox/utilities/forms/fields.py

```diff
@@ -221,6 +222,77 @@ def validate(self, value):
         return value
 
 
+class CSVFileField(forms.FileField):
```

Comment:
Most of the content of this class seems to have been duplicated from CSVDataField: We always want to minimize redundant code. Instead of duplicating the logic, we should move it out of CSVDataField into a separate class or functions that can be utilized by both CSVDataField and CSVFileField.

Reply:
I made two functions, parse_csv and validate_csv, within forms/utils.py. Both CSV fields use these functions to reduce redundancy.